### PR TITLE
Nikse.SubtitleEdit version 3.6.7.0

### PR DESCRIPTION
**Pull request by niksedk**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/85278)

### CODE REVIEW

**Comment by niksedk** [comment]
@microsoft-github-policy-service agree [company="{Nikse}"]

**Comment by niksedk** [comment]
@microsoft-github-policy-service agree company="Nikse"

File: manifests/n/Nikse/SubtitleEdit/3.6.7.0/Nikse.SubtitleEdit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Nikse.SubtitleEdit
+PackageVersion: 3.6.7.0
+PackageLocale: en-US
+Publisher: Nikse
+PackageName: Subtitle Edit
+License: GNU GPL 3
+Copyright: Copyright © 2001-2022, Nikse
+ShortDescription: Subtitle Edit Setup
```
[the remark this answers is not in the document]
Comment (KaranKad) [reply manifests/n/Nikse/SubtitleEdit/3.6.7.0/Nikse.SubtitleEdit.locale.en-US.yaml:11]: Some additional metadata

File: manifests/n/Nikse/SubtitleEdit/3.6.7.0/Nikse.SubtitleEdit.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Nikse.SubtitleEdit
+PackageVersion: 3.6.7.0
+Installers:
+- Architecture: neutral
+  InstallerType: inno
```
[the remark this answers is not in the document]
Comment (KaranKad) [reply manifests/n/Nikse/SubtitleEdit/3.6.7.0/Nikse.SubtitleEdit.installer.yaml:8]: Some additional metadata

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by KaranKad — APPROVED** [review]

**Pull request merged**